### ISSUE TITLE
iOS app stops being able to connect to server on one device but not the other (and version discrepancy?)

### ISSUE THREAD

**Issue by kikendo** [immich-app/immich#9962]
### The bug

I was running the iOS app on two devices just fine and suddenly one of them stopped being able to connect to the server.
I deleted the app and reinstalled it to no avail, that one device still does not connect to the server saying it is unreachable.
The Immich client works fine on that device on the web browser.
This happened around the time I downed the server and reupped it again.

I tried it on an ipad running iOS 16 and it worked fine.

Both phones have the latest version of iOS installed (17.5.1)

### The OS that Immich Server is running on

Debian 11

### Version of Immich Server

v1.105.1 

### Version of Immich Mobile App

v1.105.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
.
```

### Your .env content

```Shell
.
```

### Reproduction steps

```bash
1. I didn't do anything so I don't know how to reproduce exactly, I don't have any more devices to test with.
2.
3.
...
```

**Comment by schuhbacca** [comment]
Have you checked your settings to make sure it allows localhost connections in the app? I don't have an apple device but have seen this issue around.

**Comment by kikendo** [comment]
> Have you checked your settings to make sure it allows localhost connections in the app? I don't have an apple device but have seen this issue around.

Yup I quadruple-checked because I thought maybe somehow it changed? I gave it local access the first time over but yeah, the app has local network access. And it was working until yesterday before I restarted the server. I don't know if it has anything to do with that but don't see how.

**Comment by xabier-vazquez-jt** [comment]
Same here. iPhone immich app v1.105.0 and server v1.105.1 " Server is nor reachable" error.
Server is working and accessible via web.

**Comment by alextran1502** [comment]
@xabier-vazquez-jt Try restart your server with `docker compose down` then `docker compose up` make sure that you are on the same local network and/or your proxy is working if you are using one

**Comment by kikendo** [comment]
> @xabier-vazquez-jt Try restart your server with `docker compose down` then `docker compose up` make sure that you are on the same local network and/or your proxy is working if you are using one

I don't see @xabier-vazquez-jt 's response but I have other clients working just fine, it is just the one client refusing to connect via the app

**Comment by ingvardm** [comment]
Try enabling "local network" in permissions

**Comment by danieldietzler** [comment]
@kikendo did you figure it out by now?

**Comment by danieldietzler** [comment]
Every now and then really sounds like a networking issue to me tbh. It'd probably be easiest if you jump on our discord (https://discord.immich.app) and open a help desk thread there.

**Comment by kikendo** [comment]
> Every now and then really sounds like a networking issue to me tbh. It'd probably be easiest if you jump on our discord (https://discord.immich.app) and open a help desk thread there.

No, i mean every now and then, consistently, on certain clients and not on others. Clients don't randomly fall out of ability to connect, any client that works always worked.

**Comment by danieldietzler** [comment]
Ah I see. I'd still guess it's an issue with those devices, so jumping on our discord would make troubleshooting easier since it's more direct communication :)
